src/Mobs/PathFinder.cpp

```diff
@@ -0,0 +1,233 @@
+
+
+#include "Globals.h"
+#include "PathFinder.h"
+#include "../Chunk.h"
+
+cPathFinder::cPathFinder(double a_MobWidth, double a_MobHeight)
+	: m_Path(nullptr)
+	, m_GiveUpCounter(0)
+	, m_TicksSinceLastPathReset(1000)
```

Comment:
The current algorithm is somewhat like this:
- If the destination changes:
  - If TicksSinceLastPathReset > X
       Recalculate
X is a number whose value is dependent on the distance to target: The further we are from target, the larger the X. In other words: When near, recalculate often, when far, recalculate less often.

Do you have a better proposal which doesn't need `TicksSinceLastPathReset`?